src/compat.h

```diff
@@ -82,6 +82,9 @@ struct Compatibility_t
     // 1.3.6-1
     bool fix_npc_ceiling_speed; // when an NPC hits a ceiling block, it takes its SpeedY from that block, not from an arbitrary one
     bool emulate_classic_block_order; // the quadtree should return blocks in the order they had at the beginning of the level, not their current order
+    
+    bool enableSEEFeatures; //Enables certain things, like disabling the disabling of saving while cheating, exiting to the map/hub while on a level, etc.
```

Comment:
You should have separated boolean for each feature, please don't make god booleans.
- one option for anti-cheat no-save punishment
- another option for exiting to the world map/hub while on level
- ensure all combinations of both options works properly